### ISSUE TITLE
feat: edit model response (guiding model to a certain direction)

### ISSUE THREAD

**Issue by Notarin** [ollama-webui/ollama-webui#69]
This is very simple, the ability to edit the models responses like in oobabooga, or more preferably something akin to the way you edit your prompts

**Comment by tjbck** [comment]
Hi, I'm not sure If I'm getting what you're saying. could you elaborate a bit more? Thanks.

**Comment by Notarin** [comment]
sure, the prompts i provide to the model, next to each one is an edit button, I would like there to be one next to the models responses as well, as the chat history is plain text and entirely mutable, so I would like to be able to edit its response, this is useful for what you want to guide the model in a direction, or assist it come to the right conclusion. Ive used this on playground and on oobabooga's UI and it's very useful.

**Comment by guidevops** [comment]
Indeed, a necessary feature. This is useful for teaching the model how to behave.

**Comment by tjbck** [comment]
Hi all,

I'll think of ways to implement this after https://github.com/jmorganca/ollama/pull/991 gets merged to main.

Thanks.

**Comment by guidevops** [comment]
The PR was merged.

**Comment by tjbck** [comment]
Actively working on this feature! I will add this long-awaited support right after the Ollama team releases the new version with the changes. Stay tuned, thanks!
